app/src/main/java/com/kickstarter/services/transformers/GraphQLTransformers.kt

```diff
@@ -417,7 +417,7 @@ fun userTransformer(user: fragment.User?): User {
     val avatar = Avatar.builder()
         .medium(user?.imageUrl())
         .build()
-    val chosenCurrency = user?.chosenCurrency()
+    val chosenCurrency = user?.chosenCurrency() ?: "USD"
```

Comment:
Also fixed this chosenCurrency in the userTransformer that didn't have a default value

Reply:
probably is better to use instead of "USD", CurrencyCode.USD.rawValue() as default value, is not a value likely gonna change but to not have hardcoded strings around if avoidable